src/functTest/groovy/com/bmuschko/gradle/docker/DockerWorkflowFunctionalTest.groovy

```diff
@@ -314,8 +317,8 @@ class DockerWorkflowFunctionalTest extends AbstractFunctionalTest {
     }
 
     def "Can build an image only once"() {
-        File imageDir = temporaryFolder.newFolder('images', 'minimal')
-        createDockerfile(imageDir)
+        File imageDir = new File(projectDir, 'images/minimal')
```

Comment:
I still don't get this. Every test cases creates a new test folder via `temporaryFolder.newFolder` so it shouldn't matter whether a previous test failed or not. Am I missing something here?